docker/dockerfiles/be/be-ubi.Dockerfile

```diff
@@ -33,22 +33,24 @@ RUN yum install -y java-1.8.0-openjdk-devel tzdata openssl curl vim ca-certifica
     yum remove -y mysql57-community-release-el7
 ENV JAVA_HOME=/usr/lib/jvm/java-openjdk
 
+RUN touch /.dockerenv
+
 WORKDIR $STARROCKS_ROOT
 
+# Run as starrocks user
+ARG USER=starrocks
+ARG GROUP=starrocks
+RUN groupadd --gid 1000610000 $GROUP && useradd --no-create-home --uid 1000610000 --gid 1000610000 \
```

Comment:
keep uid/gid 1000 for now. need some additional fix on start/stop scripts and fe/be for openshift adaption.